api/omb_eregs/settings.py

```diff
@@ -35,7 +36,7 @@
 
 # Application definition
 
-INSTALLED_APPS = (
+INSTALLED_APPS: Tuple[str, ...] = (
```

Comment:
These kinds of annotations are needed because we're using tuples as immutable lists here, not as data types whose dimensions and types need to remain constant.